server/index.js

```diff
@@ -13,6 +15,52 @@ const db = new pg.Pool({
 });
 
 const app = express();
+const server = http.createServer(app);
+const io = new Server(server);
+
+io.on('connection', socket => {
+  const { gameId } = socket.handshake.query;
+
+  if (gameId) {
+    const room = gameId.toString();
+    socket.join(room);
+    const sql = `
+    select *
+      from "games"
+    where "gameId" = $1
+    `;
+    const params = [gameId];
+    db.query(sql, params)
+      .then(result => {
+        const meta = result.rows[0];
+        io.to(room).emit('room joined', meta);
+      })
+      .catch(err => {
+        console.error(err);
+        socket.disconnect();
+      });
+  }
+
+  socket.on('join lobby', () => {
+    socket.join('lobby');
+  });
+
+  socket.on('game joined', () => {
+    const sql = `
+    select *
+      from "games"
+     where "opponentName" is null
+    `;
+    db.query(sql)
+      .then(result => {
+        socket.broadcast.to('lobby').emit('game joined', result.rows);
+      })
+      .catch(err => {
+        console.error(err);
+        socket.disconnect();
+      });
+  });
```

Comment:
You should be able to remove this. if `gameId` is truthy, that means that a game was joined. broadcast which game was joined to the lobby after line 36